petcare-sample/android-mobile-application/README.md

```diff
@@ -0,0 +1,266 @@
+# API Authentication Android Sample - WSO2 Identity Server
+
+## ⚠️ Read this first
+
+1. Required versions
+ ```
+ Java version >= 17
+ ```
+ ```
+ Android Studio version >= Giraffe
+ ```
+
+#### 1. Setup steps Identity Server
+1.1 Setup the latest version of WSO2 Identity Sever.
+
+1.2 Create a mobile application (Un tick the PKCE `Mandayory` checkbox)
```

Comment:
Suggestion: Linking doc references to these steps would make it more convenient to a first time user to follow the readme.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/443/commits/a0de749a7604845679f1995fc79c5815c95f7a5d